src/ChoiceGroup/components/Feedback.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+
+import defaultTheme from "../../defaultTheme";
+import { TYPE_OPTIONS } from "../consts";
+import type { Props } from "./Feedback.js.flow";
+
+export const StyledFormFeedback = styled(({ theme, type, ...props }) => <div {...props} />)`
+  color: ${({ theme, type }) =>
+    type === TYPE_OPTIONS.ERROR ? theme.orbit.colorTextError : theme.orbit.colorTextSecondary};
+  font-family: ${({ theme }) => theme.orbit.fontFamily};
+  font-size: ${({ theme }) => theme.orbit.fontSizeFormFeedback};
+  font-weight: ${({ theme, type }) =>
+    type === TYPE_OPTIONS.ERROR ? theme.orbit.fontWeightMedium : theme.orbit.fontWeightNormal};
+  line-height: ${({ theme }) => theme.orbit.lineHeightText};
+  width: 100%;
+  margin-top: 2px;
+  position: absolute;
+  top: 100%;
+  max-height: ${({ theme }) =>
```

Comment:
I was also thinking if it would be worthy to use `Truncate` component? 🤔

Reply:
We might, but to me this is not related to this PR, I'm just moving the old logic to different place here, IMO we should solve this in second PR